common/src/main/java/com/wynntils/models/activities/markers/ActivityMarkerProvider.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.activities.markers;
+
+import com.wynntils.core.components.Managers;
+import com.wynntils.features.combat.ContentTrackerFeature;
+import com.wynntils.models.beacons.type.BeaconColor;
+import com.wynntils.models.marker.type.MarkerInfo;
+import com.wynntils.models.marker.type.MarkerProvider;
+import com.wynntils.models.marker.type.StaticLocationSupplier;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.mc.type.Location;
+import com.wynntils.utils.render.Texture;
+import com.wynntils.utils.type.Pair;
+import java.util.stream.Stream;
+
+public class ActivityMarkerProvider implements MarkerProvider {
+    private Pair<Location, MarkerInfo> spawnInfo;
+    private Pair<Location, MarkerInfo> trackedActivityInfo;
+
+    public void setSpawnLocation(Location spawnLocation) {
+        this.spawnInfo = spawnLocation == null
+                ? null
+                : Pair.of(
+                        spawnLocation,
+                        new MarkerInfo(
+                                new StaticLocationSupplier(spawnLocation),
+                                Texture.QUESTS_BUTTON,
+                                CommonColors.WHITE,
+                                CommonColors.WHITE));
+    }
+
+    public Location getSpawnInfo() {
+        return spawnInfo.a();
```

Comment:
Either the name needs changing or this returns the wrong thing..?

Reply:
Yes.